handlers/BaseHandler.cfc

```diff
@@ -8,6 +8,40 @@
 * to produce RESTFul responses.
 */
 component extends="coldbox.system.EventHandler"{
+	property name="APIResponse" inject="Response";
```

Comment:
Oh, when handler caching is turned on...  Yikes, I hadn't thought about that.  I'll make that change in both the REST template.